### ISSUE TITLE
Zeal version  number in the About Zeal dialog needs to be updated.

### ISSUE THREAD

**Issue by RobertinoValue** [zealdocs/zeal#246]
Not a high priority.
The Zeal version number/date in ```zeal/zeal.pro``` needs to be updated.
Zeal version ```20141123``` displays ```20140215```.

https://github.com/jkozera/zeal/blob/7112812ddc08f0cf06e1a629c739d4e61e28a62e/zeal/zeal.pro#L71

**Comment by jkozera** [comment]
It's by no means a complete fix, but the [recent automated Shippable builds](https://bitbucket.org/jerzykozera/zeal-win32-binary-downloads/) should display correct YYYYMMDD version numbers (they're automatically replaced in https://github.com/jkozera/zeal/blob/master/shippable.yml#L30)

Ideally I guess we should have proper version numbers and perhaps autodetect current one using git tags.

**Comment by RobertinoValue** [comment]
OK, I checked and noticed that the recent automated shippable builds start from 26th November.
And grabbed the latest from today. Thanks.

And I don't know if you were asking about Git tagging and versioning, but this is how I would do it.
You only need this :
```Bash
# Create annotated tags with a tagging message :
$ git tag -a v1.0.0 -m 'The famous version 1.0.0'
```
and :
```Bash
# Drop the "v" prefix. Then properly sort the version numbers. And only show the last one. Store the version number in a variable.
$ ZEAL_VERSION=$(git tag -l | cut -c2- | sort -t. -n -k1,1 -k2,2 -k3,3 -k4,4 | tail --lines=1)
```
This is the long story :

```Bash
# Create annotated tags with a tagging message :
$ git tag -a v1.0.0 -m 'The famous version 1.0.0'


# After a while :
$ git tag -l
v1.0.0
v1.0.1
v1.0.2
v1.1.0
v1.2.0
v1.2.1
v2.0.0
v2.1.0
v2.10.0
v2.11.0
v2.11.1
v2.12.0
v2.13.0
v2.13.1
v2.14.0
v2.15.0
v2.2.0
v2.3.0
v2.3.1
v2.4.0
v2.5.0
v2.6.0
v2.7.0
v2.7.1
v2.8.0
v2.9.0


# Drop the "v" prefix. Then properly sort the version numbers.
$ git tag -l | cut -c2- | sort -t. -n -k1,1 -k2,2 -k3,3 -k4,4
1.0.0
1.0.1
1.0.2
1.1.0
1.2.0
1.2.1
2.0.0
2.1.0
2.2.0
2.3.0
2.3.1
2.4.0
2.5.0
2.6.0
2.7.0
2.7.1
2.8.0
2.9.0
2.10.0
2.11.0
2.11.1
2.12.0
2.13.0
2.13.1
2.14.0
2.15.0


# Drop the "v" prefix. Then properly sort the version numbers. And only show the last one.
$ git tag -l | cut -c2- | sort -t. -n -k1,1 -k2,2 -k3,3 -k4,4 | tail --lines=1
2.15.0

# Drop the "v" prefix. Then properly sort the version numbers. And only show the last one. Store the version number in a variable.
$ ZEAL_VERSION=$(git tag -l | cut -c2- | sort -t. -n -k1,1 -k2,2 -k3,3 -k4,4 | tail --lines=1)

$ echo $ZEAL_VERSION
2.15.0
```

**Comment by trollixx** [comment]
Closing this as a duplicate of #203.

**Comment by RobertinoValue** [comment]
How can this be a duplicate of (#)203?
I did not ask to change the version numbering, like was done in (#)203.

I just pointed out that Zeal version 20141123 still displayed 20140215.
Frankly I can't be bothered what type of version numbering is used.

The code I posted was in response to the casual remark/request (last line of his comment) by jkozera.
Besides (#)203 was closed when it showed up in my search results. I didn't open this issue until after lunch from work. Hours later.
I've removed the code to avoid further confusion.

**Comment by trollixx** [comment]
@RobertinoValue there was no confusion. We need to improve the code, that generates version number. Getting one from the git tags is the best option. So, the proper fix for this issue would be implemented together with #203.

Anyway, we can keep this one open until a new release.

**Comment by schlamar** [comment]
> It's by no means a complete fix, but the recent automated Shippable builds should display correct YYYYMMDD version numbers

Nope, 20150112054614 still shows 20140215.

**Comment by schlamar** [comment]
This is hard coded in src.pro: https://github.com/zealdocs/zeal/blob/86971e90a5eeda66425efd49223320536fb1e0e5/src/src.pro#L21

**Comment by trollixx** [comment]
Fixed in 022d4be6e0f354b610d6fca60e919733a9100e59 and 8e3d3f003a223eaf69ce42a8a0ead01fedeabe18 (for Windows automatic builds).

**Comment by RobertinoValue** [comment]
Thanks @schlamar.
(Not so duplicate after all.)
